compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/BaseComposeScene.skiko.kt

```diff
@@ -74,7 +75,7 @@ internal abstract class BaseComposeScene(
         private set
 
     private var isInvalidationDisabled = false
-    private inline fun <T> postponeInvalidation(crossinline block: () -> T): T {
+    private inline fun <T> postponeInvalidation(tag: String, crossinline block: () -> T): T = trace("BaseComposeScene:postponeInvalidation($tag)") {
```

Comment:
```suggestion
    private inline fun <T> postponeInvalidation(traceTag: String, crossinline block: () -> T): T = trace(traceTag) {
```

I am not sure about concatenation performance each frame, but to be safe I would refactor. The calling code will be also more readable (subjectively):

```
override fun setContent(content: @Composable () -> Unit) = postponeInvalidation("BaseComposeScene:setContent") {

override fun render(canvas: Canvas, nanoTime: Long)  = postponeInvalidation("BaseComposeScene:render") {
```

Reply:
I doubt it has significant performance hit, but indeed, let's not degrade it.

Reply:
Plus indeed, it adds redundant double trace. All `postponeInvalidation` use places use it for entire scope

Reply:
Done